solenv/clang-format/excludelist

```diff
@@ -448,6 +448,10 @@ bridges/source/cpp_uno/gcc3_linux_aarch64/callvirtualfunction.cxx
 bridges/source/cpp_uno/gcc3_linux_aarch64/callvirtualfunction.hxx
 bridges/source/cpp_uno/gcc3_linux_aarch64/cpp2uno.cxx
 bridges/source/cpp_uno/gcc3_linux_aarch64/uno2cpp.cxx
+bridges/source/cpp_uno/gcc3_linux_loongarch64/abi.cxx
+bridges/source/cpp_uno/gcc3_linux_loongarch64/abi.hxx
+bridges/source/cpp_uno/gcc3_linux_loongarch64/cpp2uno.cxx
+bridges/source/cpp_uno/gcc3_linux_loongarch64/uno2cpp.cxx
```

Comment:
The list is sorted alphabetically, you should add the lines somewhere below.

Reply:
Actually, new code *should* get properly formatted.

During local testing and tweaking of this code, I noticed an awful lot of style inconsistencies, so you'd be better off just `clang-format`ting all of the files.